src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -10,27 +10,34 @@
  */
 public class CommandResult {
 
-    /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    /**
+     * The feedback message to be shown to the user. Contains a description of
+     * the execution result
+     */
+    private final String feedbackToUser;
 
     /** The list of persons that was produced by the command */
     private final List<? extends ReadOnlyPerson> relevantPersons;
 
     public CommandResult(String feedbackToUser) {
-        this.feedbackToUser = feedbackToUser;
-        relevantPersons = null;
+	this.feedbackToUser = feedbackToUser;
+	relevantPersons = null;
     }
 
     public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relevantPersons) {
-        this.feedbackToUser = feedbackToUser;
-        this.relevantPersons = relevantPersons;
+	this.feedbackToUser = feedbackToUser;
+	this.relevantPersons = relevantPersons;
     }
 
     /**
      * Returns list of persons relevant to the command command result, if any.
      */
     public Optional<List<? extends ReadOnlyPerson>> getRelevantPersons() {
-        return Optional.ofNullable(relevantPersons);
+	return Optional.ofNullable(relevantPersons);
+    }
+
+    public String getFeedbackToUser() {
+	return feedbackToUser;
     }
 
 }
```

Comment:
Looks like this time you fixed all the formatting to adhere to the coding style. nice!